new-frontend/src/app/helpers/task.helper.ts

```diff
@@ -44,3 +44,7 @@ export const getStatusInfoTypeForStatus = (status: TaskStatus): 'pending' | 'act
       return 'pending';
   }
 };
+
+export const isEqualString = (s1: string, s2: string) => {
+  return s1.toLowerCase() === s2.toLowerCase();
+};
```

Comment:
maybe this can be moved to more generic helper functions file (as it is not specific to tasks)? Not sure if we already have that